### ISSUE TITLE
4.10 Carthage UA scaling is off

### ISSUE THREAD

**Issue by randomnublet** [LoneGazebo/Community-Patch-DLL#11042]
From 100/25 in ancient/classical to 400/100 in Medieval, where it should be 200/50 (400/100 being the Industrial Era expected value instead).

![civbugdido2](https://github.com/LoneGazebo/Community-Patch-DLL/assets/53142313/71f03545-b37d-4c9b-bcc1-0fb28aaadb8c)

**Comment by Hokath** [comment]
As I commented in the thread: is it possible you got 2 increases in the same turn?

**Comment by randomnublet** [comment]
![civbugdido3](https://github.com/LoneGazebo/Community-Patch-DLL/assets/53142313/ab7a538c-8c82-4519-b924-aebb0ec5ce29)

No, multiple new luxuries give multiple notifications: here is the notification log after I build the Great Cothon in classical era. After the Cothon I didn't connect any new lux until Medieval, I don't know if the bug is related to the UNW (the first screenshot though, shows a new luxury I connected after conquering an enemy city, no more lux in Carthage).

**Comment by azum4roll** [comment]
Just revert 3b0c5b2. It must've been already scaling before.

**Comment by axatin** [comment]
Yes, all types of instant yields scale unless explicitly excluded.
```
//Exclusion for birth yields and GP expense and policy unlocks (as we do it up above to avoid % growth and religion bonuses from being scaled).
if (bEraScale && iType != INSTANT_YIELD_TYPE_BIRTH && iType != INSTANT_YIELD_TYPE_GP_USE && iType != INSTANT_YIELD_TYPE_POLICY_UNLOCK && iType != INSTANT_YIELD_TYPE_BORDERS && iType != INSTANT_YIELD_TYPE_REMOVE_HERESY && iType != INSTANT_YIELD_TYPE_VICTORY && iType != INSTANT_YIELD_TYPE_VICTORY_GLOBAL)
{
				iValue *= iEra;
}
```
